src/workerd/io/io-channels.h

```diff
@@ -151,7 +151,8 @@ class IoChannelFactory {
       const ActorIdFactory::ActorId& id,
       kj::Maybe<kj::String> locationHint,
       ActorGetMode mode,
-      SpanParent parentSpan) = 0;
+      SpanParent parentSpan,
```

Comment:
nit: can the SpanParent remain as the last parameter?  Our style seems to be that observability stuff goes at the end after all functionality-related parameters.

Reply:
Funnily enough I actually did this at first, then swapped the parameters right before I pushed.

Reply:
Should have trusted your gut!